src/plugins/Polls/Services.tsx

```diff
@@ -0,0 +1,187 @@
+import Gun from 'gun'
+import 'gun/lib/then.js'
+import { Result, Err, Ok } from 'ts-results'
+import { gunServers } from '../../network/gun-servers'
+import type { PollMetaData } from './types'
+import { PluginMessageCenter } from '../PluginMessages'
+
+const gun = new Gun(gunServers)
+
+interface NewPollProps {
+    sender?: string | undefined
+    id?: string | undefined
+    question: string
+    options: Object
+    start_time: Date
+    end_time: Date
+}
+
+enum Reason {
```

Comment:
Did you really use this Enum? It seems like it is copied from the Gitcoin plugin